src/lib/tvgPictureImpl.h

```diff
@@ -156,6 +156,13 @@ struct Picture::Impl
         return true;
     }
 
+    bool preserveAspectRatio(bool* preserve) const
+    {
+        if (!loader) return false;
```

Comment:
default value is true. (not false)

Reply:
I know :) but  it's the value indicating success or failure of the api, it's not the info about the aspect ratio preservation

Reply:
For consistency, c++ getter apis doesn't use return type but we prefer the immediate value return.
If you think it's a bit useful or necessary, instead we can put TVGERR() in that case with the useful info message.
for instance)
TVGERR("Picture", "Loader is invalid, Please load the file first....");